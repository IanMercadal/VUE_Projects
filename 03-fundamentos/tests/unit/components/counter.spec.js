import {shallowMount} from '@vue/test-utils'
import Counter from '@/components/Counter'

describe('Counter Component', () => {
    test('Debe ser igual a la captura' , () => {

        const wrapper = shallowMount(Counter)

        expect(wrapper.html()).toMatchSnapshot()
    })
})