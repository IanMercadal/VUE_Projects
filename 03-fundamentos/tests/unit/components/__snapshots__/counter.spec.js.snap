// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter Component Debe ser igual a la captura 1`] = `
<h2>Counter</h2>
<p>100 <sup>2</sup> = 10000</p>
<div>
  <!-- <button v:on-click="increase">+1</button>
      <button v:on-click="decrease">-1</button> --><button>+1</button><button>-1</button>
</div>
`;
